.travis.yml

```diff
@@ -16,6 +16,10 @@ language: bash
 services:
 - docker
 
+before_install:
+  - gem install package_cloud
+  - pip install --user travispy
```

Comment:
line 21 is not used, this was used to synchronise the runtest task with the builders.